Hangsman/game.py

```diff
@@ -0,0 +1,42 @@
+from hangman import Hangman
+from word_pick import Word_Picker
+
+def start():
+    choose_word = Word_Picker()
```

Comment:
since start is a static func, in future if we want to change Word_Picker logic, we would need to change this code. 
we could have create this as a class and injected Work_Picker. 
Also unit testing becomes easier with dependency injection.